generator-google-docs-addon/generators/app/templates/gapps.config.sample.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This sample is added so that generated projects still have a sample config that can be deployed to version control.